tests/functional/test_upload_service.py

```diff
@@ -25,54 +25,142 @@ def __init__(self, *args, **kwargs):
         self.db_session_maker = DBSessionMaker()
 
     def setUp(self):
+        _start_time = time.time()
+        self.api_url = f"https://{os.environ['API_HOST']}/v1"
         self.upload_config = UploadConfig()
-        self.deployment_stage = os.environ['DEPLOYMENT_STAGE']
         self.auth_headers = {'Api-Key': self.upload_config.api_key}
-        self.api_url = f"https://{os.environ['API_HOST']}/v1"
+        self.deployment_stage = os.environ['DEPLOYMENT_STAGE']
+        self.upload_area_uuid = "deadbeef-dead-dead-dead-%012d" % random.randint(0, 999999999999)
         self.verbose = True
+        _end_time = time.time()
+        print(f"Total startup time: {_end_time - _start_time} seconds.")
```

Comment:
What is the value in recording setup time?

Reply:
Just for fun. In case this ever become a bottleneck. Happy to remove it if you find it cloying.

Reply:
Fine to keep 🙂